dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
> BTW another thing, shouldn't we redirect only 404s? What if a new identifier appears that by the new convention has the same URL as a redirect?

I created those redirects using the current modules. One would need to delete something and readd it under the same name (possibly varying the caps).
I guess then it would still be the correct page, but let me quickly think about this.